### PR TITLE
Add cavm saiserver

### PR DESCRIPTION
**Pull request by NStetskovych**
Saiserver for cavium platform

### CODE REVIEW

File: dockers/docker-saiserver-cavm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+FROM docker-base
+
+RUN apt-get update \
+    && apt-get install -y libsensors4-dev \
+    python \
```
Comment (marian-pritsak) [inline dockers/docker-saiserver-cavm/Dockerfile]: Python is installed in docker-base

File: dockers/docker-saiserver-cavm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    python-dev
```
Comment (marian-pritsak) [inline dockers/docker-saiserver-cavm/Dockerfile]: Why dev package is needed?

File: dockers/docker-saiserver-cavm/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+COPY deps /root/deps
+
+RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; \
```
Comment (marian-pritsak) [inline dockers/docker-saiserver-cavm/Dockerfile]: dpkg_apt is not used anymore
Please see this [example](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-database/Dockerfile.template) on how to use SED_DPKG and this [example](https://github.com/Azure/sonic-buildimage/blob/master/rules/docker-database.mk) on how to integrate a docker image into build system

Comment (NStetskovych) [inline dockers/docker-saiserver-cavm/Dockerfile]: ths, just tested in old version.

File: dockers/docker-saiserver-cavm/Dockerfile (hunk 2 above)
Comment (NStetskovych) [inline dockers/docker-saiserver-cavm/Dockerfile]: in old case will need by thrift as far as I remember :)

**Pull request merged**